app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        person_info = list(person.values())
+        person_inst = Person(person_info[0], person_info[1])
+        if person_info[2] and list(person.keys())[2] == "wife":
+            person_inst.wife = person_info[2]
+        elif person_info[2] and list(person.keys())[2] == "husband":
+            person_inst.husband = person_info[2]
+    for name in Person.people:
```

Comment:
Iterate through the people list from the function input and create the required attributes in this loop.